recipe-client-addon/test/browser/browser_ActionSandboxManager.js

```diff
@@ -79,28 +79,23 @@ add_task(async function testError() {
       await manager.runAsyncCallback("testCallback");
       ok(false, "runAsnycCallbackFromScript throws errors when raised by the sandbox");
     } catch (err) {
-      equal(err.message, "WHY", "runAsnycCallbackFromScript clones error messages");
+      is(err.message, "WHY", "runAsnycCallbackFromScript clones error messages");
     }
   });
 });
 
 add_task(async function testDriver() {
   const script = `
     registerAsyncCallback("testCallback", async function(normandy) {
-      return normandy;
+      return Object.keys(normandy);
     });
   `;
 
   await withManager(script, async manager => {
-    const sandboxDriver = await manager.runAsyncCallback("testCallback");
+    const sandboxDriverKeys = await manager.runAsyncCallback("testCallback");
     const referenceDriver = new NormandyDriver(manager);
-    equal(
-      sandboxDriver.constructor.name,
-      "NormandyDriver",
-      "runAsyncCallback passes a driver as the first parameter",
-    );
-    for (const prop in referenceDriver) {
-      ok(prop in sandboxDriver, "runAsyncCallback passes a driver as the first parameter");
+    for (const prop of Object.keys(referenceDriver)) {
+      ok(sandboxDriverKeys.includes(prop), `runAsyncCallback's driver has the "${prop}" property.`);
```

Comment:
I should've called this out, my bad. The old test wasn't working, and I was confused about it, but figured it out while writing this comment. :D 

Values returned from an async callback in the sandbox are cloned back into the privileged context, but the driver has several functions, and when we clone the result, we don't pass the option to clone functions, so they get removed from the driver. Returning a list of keys that are available within the sandbox gets around this, and to some extent is a more accurate test anyway (since it explicitly tests what the sandbox can see, not what the value returned by the sandbox has).